explainer.md

```diff
@@ -1,751 +1,423 @@
+```css
+::page-transition-container(foo)::image-wrapper::outgoing-image {
+  /* … */
+}
+```
 
-When capturing as a 'computed style + content image', we capture the combination
-of the element's children as image, clipped to the content box. Again this is
-different to `element()`.
+We have proposed a new combinator to make it easier to select descendant pseudo elements https://github.com/w3c/csswg-drafts/issues/7346.
 
-However, these variations could be included in `element()` using modifiers or
-similarly named functions (e.g. `element-children()`).
-[Here](https://jsbin.com/bisoleziyi/edit?html,output) is a polyfill example of a
-single image mode transition built using the existing element() support in
-Firefox.
+```css
+::page-transition-container(foo) :>> outgoing-image {
+  /* … */
+}
+```
 
-Note that we are not currently planning on exposing the stand-alone method of
-capturing the elements, as would be possible with an `element()` function.
-Instead, the capturing and displaying the contents via the pseudo elements is an
-internal detail.
+This will play well with CSS nesting:
+
+```css
+::page-transition-container(foo) {
+  & :>> outgoing-image {
+    /* … */
+  }
+  & :>> incoming-image {
+    /* … */
+  }
+}
+```
+
+## Transitions targeted to a specific element
+
+In the current design, the transition acts across the whole document. However, developers have expressed interest in using this system, but limited to a single element. For example, allowing two independent components to perform transitions.
+
+This is being discussed in https://github.com/WICG/shared-element-transitions/issues/52.
 
 # Security/Privacy Considerations
 
 The security considerations below cover same-origin transitions.
 
-- Script can never read pixel content in the images. This is necessary since the
-  document may embed cross-origin content (iframes, CORS resources, etc.) and
-  multiple restricted user information (visited links history, dictionary used
-  for spell check, etc.)
-- If an element is captured as a 'computed style + content image', any external
-  resources specified on the container, such as background images, will be
-  re-fetched in the context of the new page to account for differences in
-  sandboxing.
+- Script can never read pixel content in the images. This is necessary since the document may embed cross-origin content (iframes, CORS resources, etc.) and multiple restricted user information (visited links history, dictionary used for spell check, etc.)
+- If an element is captured as a 'computed style + content image', any external resources specified on the container, such as background images, will be re-fetched in the context of the new page to account for differences in sandboxing.
 
-Cross-origin transitions aren't yet defined, but are likely to be heavily
-restricted.
+Cross-origin transitions aren't yet defined, but are likely to be heavily restricted.
 
 # Interactivity and accessibility
```

Comment:
Not sure if we should add this now since the details need to be sorted in the spec but one more relevant point:

* The duration for which DOM rendering is suppressed, to allow an author to asynchronously switch to the new DOM, input processing is also paused. This is necessary since the visual state presented to the user is inconsistent with the DOM state used for hit-testing.

Reply:
Sent another PR for this.